pygazebo/pygazebo.cc

```diff
@@ -507,6 +507,19 @@ void CloseWithoutModelbaseFini() {
   gazebo_sensor_initialized = false;
 }
 
+void CloseWithoutFini() {
```

Comment:
what is this used for?

Reply:
Was trying something due to error during closing env, not used any more, removed.